guides/source/api_app.md

```diff
@@ -222,7 +222,7 @@ section of the Rack guide for further information on them.
 
 Other plugins, including Active Record, may add additional middleware. In
 general, these middleware are agnostic to the type of application you are
-building, and make sense in an API-only Rails application.
+building and make sense in an API-only Rails application.
```

Comment:
Undo this change per our docs http://guides.rubyonrails.org/v5.0/api_documentation_guidelines.html#oxford-comma